arkane/statmech.py

```diff
@@ -1746,7 +1762,44 @@ def getFrequencies(self):
         get the frequencies corresponding to the internal rotors
         this is done by projecting their frequencies out of the force constant matrix
         """
-        return projectRotors(self.conformer, self.F, [(self.calcPath,self.pivots,self.tops,self.sigmas,self.semiclassical)], self.isLinear, self.isTS,getProjectedOutFreqs=True)
+        Es = self.Es
```

Comment:
I'd like to learn what this does: changing Es will still change self.Es as well, right? Is it just the benefit of not typing `self` and shortening the code, or is there anything more sophisticated behind it?

Reply:
It's a general practice speed optimization, so the computer isn't always looking at the object to find Es.  It probably doesn't matter much here, but it's generally considered good practice.  I've removed the Es one because it now seems to only have one reference later, but I've kept the phis one because it gets referenced multiple times.